lib/service_order_listener.rb

```diff
@@ -56,15 +50,16 @@ def process_message(msg)
   rescue OrderItemNotFound
     ProgressMessage.create(
       :level   => "error",
-      :message => "Could not find OrderItem with topology_task_ref of #{msg.payload["task_id"]}"
+      :message => "Could not find OrderItem with topology_task_ref of #{topic.payload["task_id"]}"
     )
   end
 
   def default_messaging_options
     {
       :protocol   => :Kafka,
       :client_ref => CLIENT_AND_GROUP_REF,
-      :group_ref  => CLIENT_AND_GROUP_REF
+      :group_ref  => CLIENT_AND_GROUP_REF,
```

Comment:
I don't believe `:group_ref` has any value when subscribing to topics.  @bzwei

Reply:
There is no such key for `messaging_options`